mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -192,6 +193,13 @@ uploadSingle file a = do
   x <- liftIO $ LBS.readFile file
   void . send $ fencode' putObject a (toBody x) & poServerSideEncryption .~ pure sse
 
+-- | Create a conduit sink for a given S3 Address
+s3Sink :: Address -> Sink BS.ByteString AWS ()
+s3Sink a =
+  do bs <- C.await
+     lift $ maybe (pure ()) sendRequest bs
+     where sendRequest bs = void . send $ fencode' putObject a (toBody bs) & poServerSideEncryption .~ pure sse
```

Comment:
This doesn't look right, it is going to call putObject for every byte-string that comes through? You will end up with the file just being the last chunk unless I am reading this incorrectly.